client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -170,6 +171,7 @@ const mapStateToProps = ( state ) => {
 	const hasEligibilityMessages = ! (
 		isEmpty( eligibilityHolds ) && isEmpty( eligibilityWarnings )
 	);
+	const isJetpackNonAtomic = isJetpack && ! isAtomicSite( state, siteId );
```

Comment:
May be we should create a new selector isNonAtomicJetpackSite which I think might be used in more places in Yoast related changes :)
The selector can be created at wp-calypso/client/state/selectors by combining the two existing selectors.

Reply:
@jdc91 Yeah! Was just talking with Kim about thins. I think it makes sense to have that selector now. Ideally we can use strictly a plan check or user capability thought.